### PR TITLE
Video listing

### PR DESCRIPTION
**Pull request by shrushti2000**
Added  video listing page

https://user-images.githubusercontent.com/61928369/163249854-5258ee3c-d151-4f20-b632-4b85c6fa7102.mp4

Please ignore changes made in videos db

### CODE REVIEW

File: src/components/Sidebar/Sidebar.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    background-color: var(--primary-bg-color);
+}
+
+.sidebar-icon {
+    margin: 10px;
+    width: 20%;
+}
+
+.sidebar-text {
+    margin: 15px;
```
Comment (chanchal-panpaliya) [inline src/components/Sidebar/Sidebar.css:25]: use rem units.

File: src/components/Sidebar/Sidebar.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    width: 20rem;
+    min-height: calc(100vh - 14rem);
+}
+
+.sidebar-item {
+    font-size: 2rem;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    border-radius: 0px 10px 10px 0px;
```
Comment (chanchal-panpaliya) [inline src/components/Sidebar/Sidebar.css:12]: Parent-height and width are rem units.

But here, you are using px for border-radius.

I don't know if I am right or wrong, but read this: https://stackoverflow.com/questions/23991054/does-rem-make-sense-on-border-radius.

**Review by chanchal-panpaliya — COMMENTED** [review]
code looks clean with proper naming and good UI.

**Review by amirnathani9 — APPROVED** [review]
Your code is clean, dont see any changes to do.
- File structure is awesome 
- code is clean
- Ui is too good

**Pull request merged**